plugins/LadspaEffect/caps/basics.h

```diff
@@ -76,7 +79,7 @@
 
 #define MIN_GAIN .000001 /* -120 dB */
 
-/* smallest non-denormal 32 bit IEEE float is 1.18�10-38 */
+/* smallest non-denormal 32 bit IEEE float is 1.18x10^-38 */
```

Comment:
I think this change could be omitted; I realise it got corrupted before, but that should be avoidable. The problem is the encoding used for the file - the original seems to be in Latin 1, rather than UTF-8. You should be able to copy the symbol from the diff here, then save the file as UTF-8 to avoid problems in the future.

Reply:
I think i did copy paste it and it not getting fixed, don't remember what happened the last time and the reason for the changes. Will look into it.

Reply:
This doesn't get fixed. I'll wait for your response before reverting

Reply:
I reverted.

Reply:
I checked it out locally and it looked fixed to me. I think the apparent problem is just a side-effect of how GitHub renders diffs. It seems to try to use the same encoding for both versions of the file, so the UTF-8-encoded text in the new version is decoded as Latin 1 (the encoding in the old version) and results in the mojibake you saw. Except when comparing the file with an older version, it should display correctly. For example, click on any of your recent fixup commits for that file, and you won't see a problem.

To see what's going on, you can open the file in a hex editor. The original has the byte `D7` for the multiplication sign, whereas the UTF-8 encoded version has `C3 97` (encoding `U+00D7`).

I think it's fine to keep the UTF-8 version, and accept that the diff will be misleading, but if you want a neater diff you can save the file using the "ISO 8859-1" or "Windows 1252" encodings.